infra/lib/api-stack.ts

```diff
@@ -126,25 +125,20 @@ export class APIStack extends Stack {
         }
       },
     });
-
-    // add performance alarms for monitoring prod environment
-    if (this.isProd(props)) {
-      this.addDBClusterPerformanceAlarms(dbCluster, dbClusterName, slackNotification?.alarmAction);
-    }
+    this.addDBClusterPerformanceAlarms(dbCluster, dbClusterName, slackNotification?.alarmAction);
 
     //----------------------------------------------------------
     // DynamoDB
     //----------------------------------------------------------
 
     // global table for auth token management
     const dynamoConstructName = "APIUserTokens";
-    const replicationRegion = props.config.region === "us-east-1" ? "us-east-2" : "us-east-1";
     const dynamoDBTokenTable = new dynamodb.Table(this, dynamoConstructName, {
       partitionKey: { name: "token", type: dynamodb.AttributeType.STRING },
-      replicationRegions: this.isProd(props) ? [replicationRegion] : undefined,
-      replicationTimeout: this.isProd(props) ? Duration.hours(3) : undefined,
+      replicationRegions: this.isProd(props) ? ["us-east-1"] : ["us-west-2"],
```

Comment:
do we want staging replication in `us-west-2`? that's where our sandbox env is

Reply:
Oh, good point. No reason in specific, just wanted to avoid `west-1` and `east-1/2`, since have stuff there.

Since we only have 4 US regions, can we use `us-east-1` for it? IIRC its just used for Dash's production bc of CloudFront?

Reply:
Thinking to keep in the US for compliance simplification. But if that's not a concern, could use `ca-central-1`.

Reply:
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-regions

Reply:
Good thinking about the compliance - this would be an issue if we were dealing with a prod env, however we don't need to worry about compliance in staging environments, since those are isolated from prod and don't deal with real PHI.

As such, using `ca-central-1` here will be fine.